### ISSUE TITLE
format: json tag was not generated

### ISSUE THREAD

**Issue by jwangsadinata** [gunk/gunk#152]
I started with a basic `pokemon.gunk` file which looks like this:
```
package pokemon

type PokemonService interface {
    Get(GetReq) Pokemon
}

type GetReq struct {
    ID int
}

type Rarity int

const (
    NORMAL Rarity = iota
    LEGENDARY
    MYTHIC
)

type Pokemon struct {
    Name   string
    ID     int
    Rarity Rarity
}
```

Running `gunk format` on it, results in this:
```
package pokemon

type PokemonService interface {
    Get(GetReq) Pokemon
}

type GetReq struct {
    ID int `pb:"1"`
}

type Rarity int

const (
    NORMAL Rarity = iota
    LEGENDARY
    MYTHIC
)

type Pokemon struct {
    Name   string `pb:"1"`
    ID     int    `pb:"2"`
    Rarity Rarity `pb:"3"`
}
```

Seems like the `pb` tags are generated properly, but not the `json` tags.

**Comment by vishen** [comment]
I am not so sure that `gunk format` should generate the `json` tags by default. The `pb` tags are required, so it makes sense to do those. The `json` tags are only useful if you are using a generator that looks for those fields; ie: `protoc-gen-grpc-gateway`.

@mvdan do you have a differing opinion on this?

**Comment by mvdan** [comment]
I agree that these shouldn't be generated by default. Perhaps they should be added if a json generator is enabled in `.gunkconfig`, but that would only work against a list of well-known generators that encode messages as json.

I also wonder if there'd be a one-size-fits-all solution for this. Only lowercase the names? Lowercase and remove any non-alphanumeric characters? What about camelCase, which some style guides tend to prefer?

**Comment by mvdan** [comment]
> I am unsure if this is correct for all cases (or even convert).

We can assume it's fine until proven otherwise :)
